content/_releases/22/18.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Product Update - v22.18
+description: Details of the product update v22.18 from 5th of May 2022.
+layout: rel
+version: "22.18"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-05-05
+redirect_from:
+  - /releases/22.18.html
+---
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements done in this round.
+
+### Configure login attempts
+
+Now you can configure the number of failed login attempts your users can have before
+locking them out of the system. We added a new variable `MAX_LOGIN_ATTEMPTS` to the
+HELM3 secrets chart. Set this variable any reasonable number for your system, otherwise
```

Comment:
Tenant admins may set this variable according to their requirements. The default value is set to five (5).

Reply:
@KMobbs actually HELM config is not a tenant admins responsibility because this person may not has a needed access. This is a task for OPS team

Reply:
I will write Tenant administration, not specifically tenantAdmin role holder

Reply:
@khanzadyan but this is installation configuration, not per tenant